examples/02_first_level_models/plot_fir_model.py

```diff
@@ -0,0 +1,155 @@
+"""Analysis of an fMRI dataset with a Finite Impule Response (FIR) model
+=====================================================================
+
+FIR models are used to estimate the hemodyamic response non-parametrically.
+The example below shows that they're good to do statistical inference
+even on fast event-related fMRI datasets.
+
+Specifically, the so-called 'localizer' dataset proposed by Pinel is
+downloaded, then analysed with a FIR model with 3 lags.
+4 main contrasts are estimated.
+
+"""
+
+#########################################################################
+# Grab localizer data
+
+import pandas as pd
```

Comment:
If you delete the unused sections containing the functions  `make_localizer_contrasts` and  `plot_contrast`, you could put move all imports here:
```python
import numpy as np
import pandas as pd
from nistats import datasets
from nilearn import plotting
import matplotlib.pyplot as plt
```